marklogic-data-hub-central/installer/configs/hub-central.conf

```diff
@@ -0,0 +1,6 @@
+# The location of the java executable is discovered by using the PATH by default, 
```

Comment:
One other comment - can you put these files under "src/rpm"? I like to avoid a lot of root-level directories, and these files all still count as "source code".